public/examples/download/downloadApp.js

```diff
@@ -328,16 +324,21 @@
                 myChart.setOption(option, true);
         }
 
+        urls.length = 0;
 
+        var baseUrl = (version === 'IPv6') ? testPlan.baseUrlIPv6NoPort : testPlan.baseUrlIPv4NoPort;
 
-        var baseUrl = (version === 'IPv6') ? 'http://' + testPlan.baseUrlIPv6 : 'http://' + testPlan.baseUrlIPv4;
+        for (var i = 0; i < ports.length; i++) {
```

Comment:
yes ... this allows more than 6 connections to be used if needed to resolve browser limitation